packages/realm-react/tsconfig.types.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "outDir": "./dist",
```

Comment:
Consider adding https://www.typescriptlang.org/tsconfig#incremental and https://www.typescriptlang.org/tsconfig#declarationMap too.